Resources/Prototypes/_EE/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -78,15 +78,15 @@
       map: ["enum.GunVisualLayers.Mag"]
   - type: Wieldable
   - type: GunWieldBonus
-    minAngle: -19
-    maxAngle: -16
+    minAngle: -17
+    maxAngle: -22
   - type: Gun
     minAngle: 21
     maxAngle: 32
     fireRate: 5 # 300rpm in Semi & Full Auto
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)